### PR TITLE
Configuration parameters prefix - replace Manager with Controller

### LINKED ISSUE #705: Change the prefix of the configuration parameters from `Manager` to `Controller`
## Required changes
Since the bluechi `manager` was renamed to `controller` the configuration parameters should have the appropriate prefix.

### Controller Configuration
`ManagerPort` -> `ControllerPort`

### Agent Configuration
`ManagerHost` -> `ControllerHost`
`ManagerPort` -> `ControllerPort`
`ManagerAddress` -> `ControllerAddress`

This change will break current usage.

### PR DESCRIPTION
**Pull request by ygalblum**
Resolves: #705

### CODE REVIEW

**Review by mwperina — APPROVED** [review]
Once this is merged we need a follow-up patch which will rename manager to controller in all remaining text.

**Review by mkemel — APPROVED** [review]
LGTM

File: doc/docs/getting_started/multi_node.md [hunk 1]
```diff
@@ -26,23 +26,23 @@ Once the installations succeeded, BlueChi needs to be configured on both machine
...
 ```
 
 ### Configuring the managed node
 
-In the next step the managed node (the **raspberry pi** in this case) needs to be configured. The managed node needs to know where to connect to, so the settings for the [ManagerHost](../man/bluechi-agent-conf.md#managerhost-string) as well as the [ManagerPort](../man/bluechi-agent-conf.md#managerport-uint16_t) have to be set to the IP and Port of the main node. In addition, the [NodeName](../man/bluechi-agent-conf.md#nodename-string) has to be set to **pi** (as set in the **AllowedNodeNames** in the [previous section](#configuring-the-main-node)).
+In the next step the managed node (the **raspberry pi** in this case) needs to be configured. The managed node needs to know where to connect to, so the settings for the [ControllerHost](../man/bluechi-agent-conf.md#managerhost-string) as well as the [ControllerPort](../man/bluechi-agent-conf.md#managerport-uint16_t) have to be set to the IP and Port of the main node. In addition, the [NodeName](../man/bluechi-agent-conf.md#nodename-string) has to be set to **pi** (as set in the **AllowedNodeNames** in the [previous section](#configuring-the-main-node)).
```
Comment (engelmi) [inline doc/docs/getting_started/multi_node.md]:
```suggestion
In the next step the managed node (the **raspberry pi** in this case) needs to be configured. The managed node needs to know where to connect to, so the settings for the [ControllerHost](../man/bluechi-agent-conf.md#controllerhost-string) as well as the [ControllerPort](../man/bluechi-agent-conf.md#controllerport-uint16_t) have to be set to the IP and Port of the main node. In addition, the [NodeName](../man/bluechi-agent-conf.md#nodename-string) has to be set to **pi** (as set in the **AllowedNodeNames** in the [previous section](#configuring-the-main-node)).
```

**Review by engelmi — COMMENTED** [review]
Small change in the link to the man pages, otherwise LGTM

**Pull request merged**